src/shared/models/blockchain.model.ts

```diff
@@ -0,0 +1,6 @@
+export enum WalletBlockchains {
+  bitcoin = 'bitcoin',
+  stacks = 'stacks',
+}
+
+export type WalletChainTypes = keyof typeof WalletBlockchains;
```

Comment:
Was wondering if we really need this type. It seems that as is being used, just the `WalletBlockchains` type may suffice?

As far as I can see, it is used in switch statements, which can reference the enum just fine ([playground example](https://www.typescriptlang.org/play?pretty=true&ssl=17&ssc=39&pln=1&pc=1#code/KYOwrgtgBA6ghgGwcALgIQQewMYGtsAWcAliAM5QDeAUFFAEbEraalQC8UA5I86yFwA0tKGRRw8FTlzETcZIdQC+1agDMwIbCmKYQUCAE8AysBYgAJgDFN2ABT0AXLETJ0WPIRLkAlFRFkAO5MhFAOfjR0dNhwZMAuSKgYOPhEpGQAdLwspM6RUdF6ZJjIGVgA5nYARExQgbFVggluyZ5p5ADa9AC6PiJ0KgUxcc1JHqnembKSef1R5sWlFdW19WSNo+4pXuldvXODUCoq1Eam5ta2dvCJW22TGdPyPkA)).

Reply:
Good observations, `WalletChainTypes` is used in one place and I'm thinking it might scale to be used in other places as we continue to integrate bitcoin. I will look at removing.

Reply:
Ok, as you see fit, if there's a need for this type in the future it may be worth keeping around.